src/catalogue/category/cataloguePropertiesForm.component.test.tsx

```diff
@@ -29,6 +29,7 @@ describe('Catalogue Properties Form', () => {
   };
 
   beforeEach(() => {
+    vi.resetAllMocks();
```

Comment:
This is normally put into the `afterEach`.

Reply:
The snapshots don't seem to work when I put in the afterEach, thats why I put it in the beforeEach

Reply:
When putting it in the `afterEach` the snapshot doesn't expect ids anymore, they are all like `rif0` instead of a byte string; which to me is wrong

Reply:
![image](https://github.com/ral-facilities/inventory-management-system/assets/144466888/a87cfd4f-01a9-4015-9e7e-9061b239dca7)

This is with `vi.resetAllMocks()` in the afterEach. To me that does not seem right

Reply:
That is saying the snapshot contains ids like `:r1f:` but it is getting uuidv4's instead, which is what we want. Letting it updates makes these changes:
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/25115d50-2b21-4997-a2a0-5ca888e64a75)

Reply:
Yeah I just realised that - my bad I got confused with what was actually correct